apcd-cms/src/apps/submitter_renewals_listing/templates/list_submitter_registrations.html

```diff
@@ -0,0 +1,126 @@
+{% extends "standard.html" %}
+{% load static %}
+
+{% block content %}
+
+<link rel="stylesheet" href="{% static 'apcd-cms/css/table.css' %}">
+<link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
+<link rel="stylesheet" href="{% static 'admin_regis_table/css/table.css' %}">
+
+<div class="container">
+  {% include "nav_cms_breadcrumbs.html" %}
+
+  <h1>Registration Information</h1>
+  <hr />
+  <p style="margin-bottom: 30px">Current registration information on file for your organization is listed below. On the right of the screen, select the dropdown menu under “Actions” to view or renew your organization’s registration information.</p>
+  <hr />
+  <div class="filter-container">
+    <div class="filter-content">
+      <span><b>Filter by Status: </b></span>
+      <select id="statusFilter" class="status-filter" onchange="filterTableByStatus()">
+        {% for option in status_options %}
+          <option class="dropdown-text" {% if option == selected_status %}selected{% endif %}>{{ option }}</option>
+        {% endfor %}
+      </select>
+      <span><b>Filter by Organization: </b></span>
+      <select id="organizationFilter" class="status-filter org-filter" onchange="filterTableByOrganization()">
+        {% for option in org_options %}
+          <option class="dropdown-text" {% if option == selected_org %}selected{% endif %}>{{ option }}</option>
+        {% endfor %}
+      </select>
+      {% if selected_status or selected_org %}
+        <button onclick="clearSelections()">Clear Options</button>
+      {% endif %}
+    </div>
+  </div>
+  <table id="submittersTable" class="submitters-table">
+      <thead>
+          <tr>
+              {% for k in header %}
+              <th>{{k}}</th>
+              {% endfor %}
+          </tr>
+      </thead>
+      <tbody>
+          {% for r in page %}
+              <tr>
+                  <td>{{r.biz_name}}</td>
+                  <td>{{r.type}}</td>
+                  <td>{{r.location}}</td>
+                  <td>{{r.reg_status}}</td>
+                  <td>
+                    {% include "view_registration_modal.html" %}
+                    <select id='actionsSubmitterDropdown_{{r.reg_id}}' class='status-filter' onchange="openAction('{{r.reg_id}}')">
+                      <option value="">Actions</option>
+                      <option value="viewRegistration">View Record</option>
+                      <option value="renewRegistration">Renew Registration (coming soon)</option>
```

Comment:
Should this be disabled at this point? Clicking this option points to a broken link.

Reply:
Agree with this.